provision/minikube/jaeger/values.yaml

```diff
@@ -3,6 +3,7 @@ provisionDataStore:
   cassandra: false
 allInOne:
   enabled: true
+  tag: 1.36.0
```

Comment:
what does this tag attribute represent ?

Reply:
This is the version of the jaeger image to be used. I updated it to the latest version as a part of my attempt. At the end it turned out not so solve the problem and only changing the retention time solved the problem. Still, as I tested this with the new version, I wanted to keep that latest jaeger version as well.